.github/workflows/report.yml

```diff
@@ -19,7 +19,26 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  store-prev-reports:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download previous latest test report
```

Comment:
Let's do this in two parts -- do the download here, but do the upload after the next job runs, otherwise we may overwrite the last report but then today's may fail.

Reply:
Yeah, I agree its a good point. Made the changes now.